exports/goerli.json

```diff
@@ -2367,6 +2367,751 @@
           "type": "function"
         }
       ]
+    },
+    "LBPManagerFactory": {
```

Comment:
We decided to use `LBPManagerFactoryNoAccessControl` for Goerli as well, didn't we?

Reply:
See comment above